packages/logging-winston/test/index.js

```diff
@@ -119,12 +136,12 @@ describe('logging-winston', function() {
 
       new LoggingWinston(optionsWithLogName);
 
-      assert.strictEqual(fakeLoggingOptions_, optionsWithLogName);
+      assert(isSubset(fakeLoggingOptions_, optionsWithLogName));
```

Comment:
The only thing not cool about this is, if it is not a subset, we don't see a meaningful explanation of that in the npm test output. Can we stick to `deepEqual` to see that the object is in there?

```js
var loggingOptions = extend({}, fakeLoggingOptions_);
delete loggingOptions.scope;

assert.deepEqual(loggingOptions, optionsWithLogName);
```